mods/cnc/chrome/lobby-players.yaml

```diff
@@ -148,7 +148,7 @@ Container@LOBBY_PLAYER_BIN:
 							Height: 25
 							Font: Regular
 						Image@STATUS_IMAGE:
-							X: 521
+							X: 529
```

Comment:
I believe we'll need the same change in `TEMPLATE_EDITABLE_SPECTATOR` and `TEMPLATE_NONEDITABLE_SPECTATOR` too.